database_schemas/schema.sql

```diff
@@ -32,3 +44,16 @@ CREATE TABLE books (
 
 CREATE INDEX genre_id_asc ON music_albums(genre_id ASC);
 CREATE INDEX label_id_asc ON books(label_id ASC);
+
+CREATE TABLE authors (
+    id INT PRIMARY KEY,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL
+);
+
+CREATE TABLE games (
+    id INT PRIMARY KEY,
+    multiplayer VARCHAR(100),
+    last_played_at DATE,
+    FOREIGN KEY(item_id) REFERENCES items(id)
```

Comment:
- [ ] I think you should create a foreign key on `author_id` that references `authors` table.

Reply:
😃 thank you @selma-belhadj for the feedback